magento/data-migration-tool
# magento/data-migration-tool#: Set "urn:magento:module:Magento_DataMigrationTool" a namespace schema location

### Description


**Issue:**
When developer runs 

```
bin/magento dev:urn-catalog:generate
```

command to generate URNs for PhpStorm it ignores all `*.xsd` files which specified in the `*.xml` configuration files of Data Migration Tool module. 

The reason is: `xs:noNamespaceSchemaLocation` does not contain a reference to module and now it contains a path to a particular `*.xsd` file. For example:

```
...
<groups xmlns:xs="http://www.w3.org/2001/XMLSchema-instance" xs:noNamespaceSchemaLocation="../groups.xsd">
...
```

**Solution:** current pull request fixes mentioned problem by adding `urn:magento:module:Magento_DataMigrationTool` reference to `xs:noNamespaceSchemaLocation` attribute..

```
...
<groups xmlns:xs="http://www.w3.org/2001/XMLSchema-instance" 
        xs:noNamespaceSchemaLocation="urn:magento:module:Magento_DataMigrationTool:etc/groups.xsd">
...
```

<img width="1264" alt="01" src="https://user-images.githubusercontent.com/13585327/62839799-7e358b80-bc98-11e9-9a59-896fa92acdb8.png">


### Fixed Issues (if relevant)
1. magento/data-migration-tool#<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. Run `dev:urn-catalog:generate` to create a configuration file for PhpStorm "Schemas and DTDs".

2. Open "Schemas and DTDs" in PhpStorm.

**Current result:** `*.xsd` schemes of Data Migration Tool module XML configs which you use in the data migration process have been **not generated**.

**Actual result** `*.xsd` schemes of Data Migration Tool module XML configs which you use in the data migration process should be **generated**.

### Contribution checklist
 
Thank you!
